pkg/assemblers/aws-iot.go

```diff
@@ -57,7 +57,7 @@ func AssembleAWSIoTManagerService(conf config.IotAWS, caService services.CAServi
 
 	go func() {
 		lSvc.Infof("starting SQS thread")
-		sqsQueueName := fmt.Sprintf("https://sqs.%s.amazonaws.com/%s/%s", awsConnectorSvc.Region, awsConnectorSvc.AccountID, "Lamassu-IoT-SYNC-EventBridgeOutput6A8BBEEC-LaYbNuW753SC")
+		sqsQueueName := fmt.Sprintf("https://sqs.%s.amazonaws.com/%s/%s", awsConnectorSvc.Region, awsConnectorSvc.AccountID, conf.AWSBidirectionalQueueName)
```

Comment:
Can we use the current queue name as the default for backwards compatibility of this change?

Reply:
@jjrodrig I think the last commit should do this with the "default" key. @haritzsaiz Could you check it?

Reply:
Not sure, I think it is required some metadata procesing to get the default value in case of empty field.

Reply:
I have added a few changes. 

Instead of implementing our own metadata processing function I think we should delegate this in [Viper library](https://github.com/spf13/viper?tab=readme-ov-file#establishing-defaults) defaults dealing method.

I have added also the chance to add default values in each service configuration. Now, the readConfig function gets a map of default values if it is needed.